docs/index.md

```diff
@@ -0,0 +1,162 @@
+![IEKit](images/iekit.svg){ align=left }
+The Impact Evaluator Kit (IEKit) is an enhanced version of the [CoordinationKit](https://docs.autonolas.network/product/coordinationkit/), which leverages [Ceramic streams](https://developers.ceramic.network/docs/advanced/standards/stream-programs/) to automate the tracking and rewarding of users' contributions on the ecosystem. The Autonolas Impact Evaluator (IE) is an agent service based on the IEKit. This service is designed to track contributions in the form of Twitter mentions of the Autonolas DAO ([@autonolas](https://twitter.com/autonolas)). The service implements three main features:
```

Comment:
Should this be called Autonolas Impact Evaluator? Seems pretty official for something that's currently a demo. Really it's just an autonomous service which is powering the IEKit demo app.

Reply:
fixed